src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -33,11 +33,17 @@
 #include OS_HEADER_INLINE(os)
 
 const char* VM_Version::_uarch = "";
+const char* VM_Version::_vm_mode = "";
 uint32_t VM_Version::_initial_vector_length = 0;
 
 void VM_Version::initialize() {
   get_os_cpu_info();
 
+  // check if satp.mode is supported, currently supports up to SV48(RV64)
+  if (get_satp_mode() > VM_SV48) {
+    vm_exit_during_initialization(err_msg("Unsupported satp mode: %s", _vm_mode));
```

Comment:
Looks like [JDK-8301067](https://bugs.openjdk.org/browse/JDK-8301067) was missed?

Reply:
Good catch!

Reply:
> Looks like [JDK-8301067](https://bugs.openjdk.org/browse/JDK-8301067) was missed?

Thanks, I've added a commit to fix it.